data/data/gcp/network/internal-lb.tf

```diff
@@ -0,0 +1,49 @@
+resource "google_compute_address" "cluster_private_ip" {
+  name = "${var.cluster_id}-cluster-private-ip"
+
+  address      = cidrhost(var.master_subnet_cidr, -3) #Last two addressed used by gcp.
```

Comment:
is this required? can we depend on GCP to give us an available ip.

Reply:
Nope, removed.